DEVEL.md

```diff
@@ -183,34 +183,99 @@ container images with the executables under test must be available in the
 cluster. If these two conditions are satisfied, run the tests with:
 
 ```bash
-$ go test -v ./test/e2e/...
+# Run all e2e tests in this repository
+go test -v ./test/e2e/...
 ```
 
-In case you want to run only certain tests, e.g., QAT ones, run:
+If you need to specify paths to your custom `kubeconfig` containing
+embedded authentication info then add the `-kubeconfig` argument:
 
 ```bash
-$ go test -v ./test/e2e/... -args -ginkgo.focus "QAT"
+go test -v ./test/e2e/... -args -kubeconfig /path/to/kubeconfig
 ```
 
-If you need to specify paths to your custom `kubeconfig` containing
-embedded authentication info then add the `-kubeconfig` argument:
+The full list of available options can be obtained with:
 
 ```bash
-$ go test -v ./test/e2e/... -args -kubeconfig /path/to/kubeconfig
+go test ./test/e2e/... -args -help
 ```
 
-The full list of available options can be obtained with:
+In most cases, it would not be possible to run all E2E tests in one system.
+For running a subset of tests, there are labels that you can use to pick out specific parts.
+You can run the tests with:
+```bash
+# Run a subset of tests
+go test -v ./test/e2e/... -args -ginkgo.focus <some labels> -ginkgo.skip <some labels>
+```
+
+#### Table of Labels
+
+| Device | Mode             | Resource    | App                            |
+|:-------|:-----------------|:------------|:-------------------------------|
+| `dlb`  |-                 | `pf`, `vf`  | `libdlb`                       |
+| `dsa`  |-                 | `dedicated` | `accel-config`                 |
+| `fpga` | `af`, `region`   |             | `opae-nlb-demo`                |
+| `gpu`  |-                 | `i915`      | `busybox`, `tensorflow`        |
+| `iaa`  |-                 | `dedicated` | `accel-config`                 |
+| `qat`  | `dpdk`           | `dc`        | `openssl`                      |
+| `qat`  | `dpdk`           | `cy`        | `openssl`, `crypto-perf`       |
+| `qat`  | `dpdk`           | `generic`   | `crypto-perf`, `compress-perf` |
+| `qat`  | `kernel`         | `cy1_dc0`   | `busybox`                      |
+| `sgx`  |-                 |             | `sgx-sdk-demo`                 |
+
+#### Examples
```

Comment:
would you mind adding an example how to run *all* tests available with Sapphire rapids bare metal? DLB, IAA, DSA, and QAT at the same time. Perhaps with SGX enabled too since it's available.

Reply:
So, you mean:
Case 1. DLB,DSA,IAA,QAT devices
Case 2. Those devices + SGX?